### ISSUE TITLE
Opening index.html file from examples does not open network in browser.

### ISSUE THREAD

**Issue by jwscot** [jacomyal/sigma.js#1203]
Dear all,

Please forgive the simple nature of my question as I am new to web development. 

I have edited an example from your github (node-reducers) and it now creates my network very well when I run 

npm start --example=use-reducers

on my terminal, however I want to people to simply be able to open an html file and be able to interact with this network. When I open the index.html file the graph doesn't render - why is this?

Is there a way to sole this simply or do I need to be able to access the packages installed via npm to run sigma/graphology?

**Comment by jacomyal** [comment]
Hi @jwscot,

Several people now have asked for a simpler way to play with sigma, and the discussion is still going on **in ticket #1146**.
Basically, we are trying to release minified versions of sigma (and graphology, or maybe only one unique bundle) on some CDN, so that people can play with sigma without all the build shenanigans. Once it's done, you'll be able to simply load sigma from any HTML file by using the proper CDN URL, and play with it, as it's possible to do with many other libraries.

I close this ticket now, to keep this discussion in a single unique place.

**Comment by jwscot** [comment]
Hi @jacomyal ,

I understand. So, just to clarify, there is no way that my network can be opened with a single file as of yet? No more changes need to be made to my network, I just want others to be able to easily access it and see the information that it holds.

Thanks.

**Comment by jacomyal** [comment]
Ha, sorry indeed, I think I misunderstood your initial question...

There is a way to build and bundle an example with all its dependencies, running `npm run build` when in the `examples` folder., and check the new `examples/build` folder, which has the build and shareable assets of each examples.

However, it's kind of broken right now, as it's a feature we started to build for the new website that we somehow stopped working on when we found out how simple it is to get working and online-editable examples using CodeSandbox.

So, from what I just saw, here are the **actions to take** to fix this feature:

  - [ ] Update the `examples/examples.json` registry
  - [ ] When building an example, move `input/public/*` to `output/.` and not in a new `output/public` folder, to fix assets loading

I reopen this ticket and add this to the v2.2 milestone as well.
Thanks for the report!
